Orange/canvas/application/canvasmain.py

```diff
@@ -1067,7 +1067,7 @@ def ask_save_changes(self):
         document = self.current_document()
         title = document.scheme().title or "untitled"
         selected = message_question(
-            self.tr('Do you want to save the changes you made to workflow "%s"?')
+            self.tr('Do you want to save the changes you made to the workflow "%s"?')
```

Comment:
Article would be appropriate if there weren't a specific workflow name (proper noun) provided.

https://www.google.com/search?q="save+the+changes+you+made+to"